src/foam/nanos/crunch/lite/rules.jrl

```diff
@@ -205,7 +205,7 @@ p({
   "action":{
     "class":"foam.nanos.crunch.lite.ruler.UCJApprovableValidateAction"
   },
-  "enabled":true,
+  "enabled":false,
```

Comment:
Why are  we turning this off?

Reply:
This is the one I mentioned to you earlier - it impossible to save / auto-save UCJ changes that aren't valid yet. I think we need to revisit this and get it working for the approval request itself rather than individual approvables (i.e. so you can submit for approval only if your data is valid, but you can still "draft" changes to your UCJs)